zcash_client_sqlite/src/testing.rs

```diff
@@ -1058,7 +1063,8 @@ pub(crate) fn input_selector(
     standard::SingleOutputChangeStrategy,
 > {
     let change_memo = change_memo.map(|m| MemoBytes::from(m.parse::<Memo>().unwrap()));
-    let change_strategy = standard::SingleOutputChangeStrategy::new(fee_rule, change_memo);
+    let change_strategy =
+        standard::SingleOutputChangeStrategy::new(fee_rule, change_memo, ShieldedProtocol::Sapling);
```

Comment:
I think this also needs to be an argument of the test helper, if the others are?